File: 典型90/Typical90_014.py
import collections
import itertools
import math
import re
import sys
import heapq

input = sys.stdin.readline

def main():
    ans = 0
    n = int(input())
    a = list(map(int, input().split()))
    b = list(map(int, input().split()))
    a.sort()
    b.sort()
    for i in range(n):
        ans += abs(a[i] - b[i])
    print(ans)

if __name__ == '__main__':
    main()

File: 典型90/Typical90_014.rs
use proconio::input;

fn main() {
    input!{
        n : usize,
        mut a : [isize; n],
        mut b : [isize; n]
    }

    let mut ans : isize = 0;

    a.sort();
    b.sort();

    for i in 0..n {
        ans += (a[i] - b[i]).abs();
    }

    println!("{}", ans);
}